docs/user/discover.asciidoc

```diff
@@ -3,101 +3,210 @@
+- You should have an understanding of {ref}/documents-indices.html[{es} documents and indices].
 
-The first thing to do in *Discover* is to select an <<index-patterns, index pattern>>, which
-defines the data you want to explore and visualize.
-If you haven't yet created an index pattern, you can add a <<add-sample-data, sample data set>>,
-which has a pre-built index pattern.
 
 [float]
-=== Set a time filter
+[[whats-you-goal-in-discover]]
+=== Step 1. Define your goal
+
+When you explore your data in **Discover**, it's common to start with one or two goals:
+
+- **Get an overview of what is happening.**
+For example, you might look for
+information on the overall health and performance of your ecommerce business,
+and then share your findings in a report.
+
+- **Find an answer to a specific question.** You want
+to determine your customers' shopping preferences,
+and then visualize your findings on a dashboard.
 
-By default, *Discover* shows data for the last 15 minutes.
-If you have a time-based index, and no data displays,
-you might need to increase the time range. Using the <<set-time-filter, time filter>>,
-you can specify a common or recently-used time range, a relative time
-from now, or an absolute time range.
+For this tutorial, your goal is to better manage your product inventory.  You want to
+know the top-selling products and on what day of the week these products sell the most.
 
 [float]
-=== Search your data
+[[find-the-data-you-want-to-use]]
+=== Step 2. Find your data
+
+Tell {kib} where to find the data you want to explore, then specify the time range in which to view that data.
+
+. Open the main menu, and select **Discover**.
+
+. Select the data you want to work with.
++
+{kib} uses an <<index-patterns,index pattern>> to tell it where to find
+your {es} data.
+To view the ecommerce sample data, make sure the index pattern is set to **kibana_sample_data_ecommerce**.
++
+[role="screenshot"]
+image::images/discover-index-pattern.png[How to set the index pattern in Discover]
+
+. Adjust the time range to view data for the *Last 7 days*.
++
+NOTE: The range selection is based on the default time field in your data.
+If you are using the sample data, this value was set when you added the data.
+If you  are using your own data, and it does not have a time field, the range selection is not available.
 
-Now that you have your data and set the time span, you can start asking your questions.
-You can search your data using the <<kuery-query, Kibana Query language>>,
-which offers a simplified query syntax.
-For example, if
-you search for `day_of_week : Friday`, you'll get a list of all documents
-in which `day_of_week` is set to `Friday`. If you prefer
-<<lucene-query, Lucene query syntax>>, you can access it from the KQL menu.
+. To view the count of documents for a given time in the specified range,
+click and drag the mouse over the histogram, and then click *Refresh*.
```

Comment:
You shouldn't need to click refresh when you select a time range in Discover by the histogram. You click on the start of the range, drag to the end, end when you release the mouse button, it's refreshed automatically